docs/user/alerting/defining-rules.asciidoc

```diff
@@ -2,114 +2,10 @@
 [[defining-alerts]]
 == Defining rules
 
-{kib} alerting rules can be created in a variety of apps including <<xpack-apm,*APM*>>, <<xpack-ml,*{ml-app}*>>, <<metrics-app,*Metrics*>>, <<xpack-siem,*Security*>>, <<uptime-app,*Uptime*>> and from the <<management,*Management*>> UI. While alerting details may differ from app to app, they share a common interface for defining and configuring rules that this section describes in more detail.
-
-[float]
-=== Create a rule
-
-When you create a rule, you must define the rule details, conditions, and actions.
-
-. <<defining-alerts-general-details, General rule details>>
-. <<defining-alerts-type-conditions, Rule type and conditions>>
-. <<defining-alerts-actions-details, Action type and action details>>
-
-image::images/rule-flyout-sections.png[The three sections of a rule definition]
+This content has been moved to <<create-and-manage-rules>>.
```

Comment:
Had to leave this file and anchor link here temporarily to avoid bad cross-document link errors. When this PR is merged, I will create PRs in the obs and ML docs repos to update links in their docs, and then I will make another PR to clean up these placeholders.